packages/algob/src/builtin-tasks/console.ts

```diff
@@ -1,42 +1,90 @@
-// import debug from "debug";
+import repl from "repl";
+import { runInNewContext } from "vm";
 
+import { mkDeployer } from "../builtin-tasks/run";
 import { task } from "../internal/core/config/config-env";
-import { AlgobRuntimeEnv } from "../types";
+import { txWriter, TxWriterImpl } from "../internal/tx-log-writer";
+import { isRecoverableError, preprocess } from "../internal/util/console";
+import { mkAccountIndex } from "../lib/account";
+import { createAlgoOperator } from "../lib/algo-operator";
+import { loadASAFile } from "../lib/asa";
+import { createClient } from "../lib/driver";
+import { loadCheckpointsRecursive } from "../lib/script-checkpoints";
+import { AlgobDeployer, AlgobRuntimeEnv, CheckpointRepo } from "../types";
 import { TASK_CONSOLE } from "./task-names";
 
-export default function (): void {
-  // const log = debug("algob:core:tasks:console");
+// colorize text to yellow
+function colorize (message: string): string {
```

Comment:
yellow color is generally used for warning, can we change it to something else ?
or maybe white is good ?

Reply:
I think yellow color is used as `info` rather than a warning. I don't think using white is good. What other color do you suggest?

Reply:
Changing to green.